src/spork.cpp

```diff
@@ -57,6 +57,20 @@ void CSporkManager::LoadSporksFromDB()
             continue;
         }
 
+        // TODO: Temporary workaround for v5.0 clients to ensure up-to-date protocol version spork
+        if (sporkDef.sporkId == SPORK_15_NEW_PROTOCOL_ENFORCEMENT_2) {
+            LogPrintf("%s : Spork 15 signed at %d\n", __func__, pSporkDB->GetSporkTime(SPORK_15_NEW_PROTOCOL_ENFORCEMENT_2));
+            // 1578338986 is the timestamp that spork 15 was last signed at for mainnet for the previous
+            // protocol bump. If the timestamp in the DB is equal or lower than this, we know that
+            // the value is stale and should ignore it to prevent un-necessary disconnections in the
+            // version handshake process. This value is also suitable for testnet as the timestamp
+            // for this spork on that network was signed shortly after this.
+            if (pSporkDB->GetSporkTime(SPORK_15_NEW_PROTOCOL_ENFORCEMENT_2) <= 1578338986 ) {
+                LogPrintf("%s : Stale spork 15 detected, ignoring...\n", __func__);
```

Comment:
Let's not make a second database read. Better to save the value in a local variable at line 62, and reuse it here.

Reply:
the value is already in a local variable, line 54. Can directly use it.

Reply:
yep, updated. I originally had this check elsewhere but moved it here.